test/unit/transitions/multi_report_alerts.js

```diff
@@ -105,6 +105,11 @@ exports['validates config : num_reports_threshold'] = test => {
   testConfigIsValid(test, [_.omit(alert, 'num_reports_threshold')]);
 };
 
+exports['validates config : num_reports_threshold < 100'] = test => {
+  alert.num_reports_threshold = 100000000; // arbitrary large number
+  testConfigIsValid(test, [ alert ]);
```

Comment:
I don't understand this.  Is a very big `num_reports_threshold` supposed to be valid (as per the code), or invalid (as I _think_ the name of the test implies)?